Build.md

```diff
@@ -0,0 +1,123 @@
+# Build
+
+The following paragraphs describe step by step how to build `neovim-qt`.
+
+## Ubuntu 18.04
+
+Get the `neovim-qt` source code either via git
+
+    $ sudo apt install -y git`
+    $ git clone https://github.com/equalsraf/neovim-qt.git
+    $ cd neovim-qt
+
+or download the zip file.
+
+    $ sudo apt install -y wget unzip
+    $ wget -c https://github.com/equalsraf/neovim-qt/archive/master.zip
+    $ unzip master.zip
+    $ cd neovim-qt-master
+
+Navigate to the `neovim-qt` source code directory and install the necessary dependencies (Qt, `cmake, compiler and `neovim`).
+
+    $ sudo apt install -y cmake build-essential qt5-qmake qt5-qmake-bin qtbase5-dev \
+        qtbase5-dev-tools qtchooser libqt5concurrent5 libqt5core5a libqt5dbus5 \
+        libqt5gui5 libqt5network5 libqt5widgets5 libqt5xml5 neovim
+    $ mkdir build
+    $ cd build
+    $ cmake -DCMAKE_BUILD_TYPE=Release ..
+    $ make
+
+If the build was successfull, you can start `neovim-qt`.
+
+    $ set NVIM_QT_RUNTIME_PATH=../src/gui/runtime ./bin/nvim-qt
+
+## Windows 10 and MinGW
+
+Download and install [`neovim`](https://github.com/neovim/neovim/wiki/Installing-Neovim).
+
+The following commands have been tested with Powershell 5.1.
+
+Get the `neovim-qt` source code either via [git](https://git-scm.com/downloads)
+
+    > git clone https://github.com/equalsraf/neovim-qt.git
+    > cd neovim-qt
+
+or download the zip file.
+
+    > Invoke-WebRequest -Uri https://github.com/equalsraf/neovim-qt/archive/master.zip -OutFile master.zip
+    > Expand-Archive -Path master.zip .
+    > cd neovim-qt-master
+
+The easiest way to build `neovim-qt` on Windows 10 is to use the Qt installer (for example the [community edition](https://www.qt.io/download-open-source)).
+Download and run the Qt installer when asked which components to install select:
+
+- Qt _Version_
+    - MinGW _MinGW Version_
```

Comment:
super nit: A little confusing, the packages is written twice... Elsewhere too.

Maybe: `_{Version}_`?

Reply:
I thought about that, but decided initially against it to make clear, that the _MinGW Version_ can be different from the _CMake Version_. If you think this is more confusing than helping I will remove it.

Reply:
Sure, sounds good.